modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -3666,6 +4016,466 @@
+		end
+	end
+
+	function m.androidDebugInformationFormat(cfg)
+		if cfg.symbols ~= "Default" then
+
+			if cfg.symbols == p.OFF then
+				m.element("DebugInformationFormat", nil, "None")
+			elseif cfg.symbols == "Full" then
+				m.element("DebugInformationFormat", nil, "FullDebug")
+			else
+				m.element("DebugInformationFormat", nil, "LineNumber")
+			end
+
+		end
+	end
+
+	function m.androidExceptionHandling(cfg, condition)
+		-- Note: Android defaults to 'off'
+		local exceptions = {
+			On = "Enabled",
+			Off = "Disabled",
+			UnwindTables = "UnwindTables",
+		}
+		if _ACTION >= "vs2015" then
+			if exceptions[cfg.exceptionhandling] ~= nil then
+				m.element("ExceptionHandling", condition, exceptions[cfg.exceptionhandling])
+			end
+		else
+			if cfg.exceptionhandling == premake.ON then
+				m.element("GccExceptionHandling", condition, "true")
+			end
+		end
+	end
+
+	function m.androidRuntimeTypeInfo(cfg, condition)
+		-- Note: Android defaults to 'off'
+		if cfg.rtti == premake.ON then
+			m.element("RuntimeTypeInfo", condition, "true")
+		end
+	end
+
+	function m.androidWarningLevel(cfg, condition)
+
+		if _ACTION >= "vs2015" then
+			if cfg.warnings and cfg.warnings ~= "Off" then
+				m.element("WarningLevel", nil, "EnableAllWarnings")
+			else
+				m.warningLevel(cfg, condition)
+			end
+		else
+			m.warningLevel(cfg, condition)
+		end
+
+	end
+
+	function m.androidClCompilePreprocessorDefinitions(cfg, condition)
+		m.preprocessorDefinitions(cfg, cfg.defines, false, condition)
+	end
+
+	function m.androidLanguageStandardCpp(cfg)
+		local cpp_langmap = {
+			["C++98"]   = "c++98",
+			["C++11"]   = "c++11",
+			["C++14"]   = "c++1y",
+			["C++17"]   = "c++1z",
+			["C++latest"] = "c++1z",
+			["gnu++98"] = "gnu++98",
+			["gnu++11"] = "gnu++11",
+			["gnu++14"] = "gnu++1y",
+			["gnu++17"] = "gnu++1z",
+		}
+		if cpp_langmap[cfg.cppdialect] ~= nil then
+			m.element("CppLanguageStandard", nil, cpp_langmap[cfg.cppdialect])
+		end
+	end
+
+	function m.androidLanguageStandardC(cfg)
+		local c_langmap = {
+			["C98"]   = "c98",
+			["C99"]   = "c99",
+			["C11"]   = "c11",
+			["gnu99"] = "gnu99",
+			["gnu11"] = "gnu11",
+		}
+		if c_langmap[cfg.cdialect] ~= nil then
+			m.element("CLanguageStandard", nil, c_langmap[cfg.cdialect])
+		end
+	end
+
+	function m.androidAdditionalCompileOptions(cfg)
+
+		if _ACTION >= "vs2015" then
+
+		else
+			local function alreadyHas(t, key)
+				for _, k in ipairs(t) do
+					if string.find(k, key) then
+						return true
+					end
+				end
+				return false
+			end
+
+			if not cfg.architecture or string.startswith(cfg.architecture, "arm") then
+				-- we might want to define the arch to generate better code
+--				if not alreadyHas(cfg.buildoptions, "-march=") then
+--					if cfg.architecture == "armv6" then
+--						table.insert(cfg.buildoptions, "-march=armv6")
+--					elseif cfg.architecture == "armv7" then
+--						table.insert(cfg.buildoptions, "-march=armv7")
+--					end
+--				end
```

Comment:
Comment code to remove?

Reply:
Part of the original code

Reply:
I'm in favour of just dropping the pre-VS2015 support, we can save that for a future PR.